### ISSUE TITLE
Custom pomodoro sound / pomodoro up / pomodoro tick sound

### ISSUE THREAD

**Issue by 99ha66** [johannesjo/super-productivity#911]
(1) Custom Tomato Sound - This will allow you to insert a motivational speech.

(2) Please make a pop-up when Work and Break times are over. In the center of the desktop, on top of all windows.

P.S Sorry for my English.
![108194802-19c67880-7128-11eb-9e61-3608392f2e82](https://user-images.githubusercontent.com/79184811/108196804-98241a00-712a-11eb-8783-594629a4ce0e.png)

**Comment by johannesjo** [comment]
Thanks for opening this up! Makes sense to me. Also a custom tick sound was requested at some point (but I can't find the ticket :-( ).

PRs welcome!

**Comment by 99ha66** [comment]
And what do you need to make a pop-up ?

**Comment by johannesjo** [comment]
About 2. You mean to replace the already existing Dialog inside the app? I think this is not really feasible unless we have very good reasons to replace it. But I definitely like the idea to display motivational messages or stats in it!

**Comment by 99ha66** [comment]
Yes, that's what I had in mind.

Existing dialogue box:

- Without dialogue
- very small
- poorly informed
- In the corner of the screen
- Disappears fast
- Without design

It has no practical use. I suggested an improved replacement.
![Безымянный](https://user-images.githubusercontent.com/79184811/108235796-45148c00-7157-11eb-8554-07bf696a1a80.png)

**Comment by timlonggr** [comment]
Custom sounds would _definitely_ be helpful--I miss the notification often because I just don't hear it or see it.

**Comment by AnasR7** [comment]
is this issue about new features still updated or not?

**Comment by johannesjo** [comment]
@AnasR7 sorry, I don't understand! Are you asking to reopen this?

**Comment by Lippiece** [comment]
> @AnasR7 sorry, I don't understand! Are you asking to reopen this?

Hi, it would be awesome if you could use custom sounds for break/pomodoro start/end. Honestly, available sounds just go by my ear, I often realise during work that the break was supposed to happen and is now over for some time already.

**Comment by azzamsa** [comment]
Yes, If this is implemented. I would  love to use gnomepomodoro `bird chirping` sound for the tick https://github.com/gnome-pomodoro/gnome-pomodoro/tree/master/data/sounds

**Comment by Lippiece** [comment]
> This issue has not received any updates in 90 days. Please comment, if this still relevant!

A strange practice, but yes, relevant.

**Comment by manicmarvin** [comment]
Gentle boop.
